doc/changelog/11-infrastructure-and-dependencies/12425-ci+gh_actions_win.rst

```diff
@@ -0,0 +1,11 @@
+- **Changed:**
+  The Windows installer CI build has been moved from custom workers to
```

Comment:
That's not something that we typically document in the user-facing changelog (which is already long enough without these kinds of details). It would make more sense in the dev changelog.

Reply:
As you prefer, I was under the impression that some users were downloading the windows artifacts for master, so I signaled it.

Reply:
That's a good point. Maybe reformulate to put this aspect forward (and maybe do not link to the fixed issues which are not really relevant for users).

Reply:
Ok, I did tweak it.